src/base.config.ts

```diff
@@ -88,7 +89,7 @@ export default function webpackConfigFactory(args: any): webpack.Configuration {
 	const jsonpIdent = args.element ? args.element.name : 'custom-elements';
 	const extensions = args.legacy ? ['.ts', '.tsx', '.js'] : ['.ts', '.tsx', '.mjs', '.js'];
 	const compilerOptions = args.legacy ? {} : { target: 'es6', module: 'esnext' };
-	const features = args.legacy ? args.features : ['chrome'];
+	const features = args.legacy ? args.features : { ...(args.features || {}), ...getFeatures('chrome') };
```

Comment:
Ah this is actually `modern` now, missed updating this when I updated webpack-contrib

Reply:
Easily done here. Once CI passes I'll merge. Thanks!